velox/row/CompactRow.h

```diff
@@ -108,6 +118,12 @@ class CompactRow {
   /// Serializes struct value to buffer. Value must not be null.
   int32_t serializeRow(vector_size_t index, char* buffer);
 
+  // Serializes struct value to buffer. Value must not be null.
```

Comment:
Use /// for public comments. 

The comment doesn't seem to match the code. It suggests that only one row is serialized, but I believe this method serializes multiple rows. Please, revise.